--- /home/anto/Documents/ItAll/gobusybox_master/src/pkg/bb/bbmain/cmd/main.go
+++ /home/anto/Documents/ItAll/gobusybox/src/pkg/bb/bbmain/cmd/main.go
@@ -78,9 +78,9 @@
 	}
 	if errors.Is(err, bbmain.ErrNotRegistered) {
 		log.SetFlags(0)
-		log.Printf("Failed to run command: %v", err)
+		log.Printf("%v", err)
 
-		log.Printf("Supported commands are:")
+		log.Printf("\nSupported commands are:")
 		for _, cmd := range bbmain.ListCmds() {
 			log.Printf(" - %s", cmd)
 		}
